src/components/Breadcrumbs/index.tsx

```diff
@@ -1,25 +1,38 @@
 import React from 'react';
+import { VscHome } from 'react-icons/vsc';
+import { useTheme } from 'styled-components';
 
 import { Section, Divider, Main } from './styles';
 
+export const iconVariant = {
+  home: VscHome
+};
+
 export type ItemProps = {
   active?: boolean;
+  icon?: keyof typeof iconVariant;
   children: React.ReactNode;
 };
+
 export type BreadcrumbProps = {
-  size?: 'medium' | 'large';
   children: React.ReactNode;
 };
 
-const Breadcrumb = ({ children, size = 'medium' }: BreadcrumbProps) => (
-  <Main size={size}>{children}</Main>
-);
+const Breadcrumb = ({ children }: BreadcrumbProps) => <Main>{children}</Main>;
 
-const Item = ({ children, active }: ItemProps) => {
+const Item = ({ children, active, icon }: ItemProps) => {
+  const theme = useTheme();
   return (
     <>
-      <Section active={active}>{children}</Section>
-      <Divider active={active}>|</Divider>
+      <Section active={active}>
+        {icon &&
+          React.createElement(iconVariant[icon], {
+            size: 12,
+            color: active ? theme.colors.pink[100] : theme.colors.grey[100]
```

Comment:
I think it's better to add the color directly in styles, what do you think @miguel-nascimento?